test/unit/jobs/cleanup-test.js

```diff
@@ -103,6 +110,39 @@ describe("Delete unused files", () => {
     }, 200);
   });
 
+  it("should delete financial data that has not been accessed within the last 24 hours", (done) => {
+    // Wait for file to be deleted before running test.
+    setTimeout(function() {
+      cleanupJob.financial.findByKey("risefinancial_realtime_HXXHCTR5AQFQ_-Kn5MaUE2cpYN-FCQSOU_1M_", (err, foundFinancial) => {
+        expect(Object.keys(foundFinancial.data).length).to.equal(0);
+
+        done();
+      })
+    }, 200);
+  });
+
+  it("should delete spreadsheet data that has not been accessed within the last 24 hours", (done) => {
+    // Wait for file to be deleted before running test.
+    setTimeout(function() {
```

Comment:
@tejohnso I am not sure doing that will be better because at this point all the tasks on the cleanup job runs async so I think it is better than running then sync for then returning a promise or calling a callback when all of them has finished. Thoughts? cheers